web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -745,7 +745,9 @@
           if (xhr.status === 200) {
             if (dataType === "script") {
               try {
+                /* jshint ignore:start */
                 eval(xhr.responseText);
+                /* jshint ignore:end */
```

Comment:
I don't suppose there's a way to indicate this without polluting the file itself?